mingw-w64-aws-sdk-cpp/PKGBUILD

```
@@ -0,0 +1,58 @@
+# Maintainer: Andrew Sun <adsun701@gmail.com>
+
+_realname=aws-sdk-cpp
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.7.365
+pkgrel=1
+pkgdesc="AWS SDK for C++ (mingw-w64)"
+arch=('any')
+url="https://github.com/aws/aws-sdk-cpp"
+license=('Apache-2.0')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-ninja")
+options=('strip' 'staticlibs')
+source=("${_realname}-${pkgver}.tar.gz::https://github.com/aws/aws-sdk-cpp/archive/${pkgver}.tar.gz"
+        "aws-sdk-cpp-pr-1333.patch"
+        "WinHttp-missing-flags.patch"
+        "Patch-cmake-submodules.patch"
+        "BuildAwsCCommon.patch"
+        "BuildAwsChecksums.patch")
+sha256sums=('95e3f40efaea7b232741bfb76c54c9507c02631edfc198720b0e84be0ebb5e9d'
+            'SKIP'
+            'SKIP'
+            'SKIP'
+            'SKIP'
+            'SKIP')
+
+prepare() {
+  cd "${_realname}-${pkgver}"
+  patch -p1 -i "${srcdir}/aws-sdk-cpp-pr-1333.patch"
+  patch -p1 -i "${srcdir}/WinHttp-missing-flags.patch"
+  patch -p1 -i "${srcdir}/Patch-cmake-submodules.patch"
+}
+
+build() {
+  export CFLAGS="-D_WIN32_WINNT=0x0600"
+  export CXXFLAGS="-D_WIN32_WINNT=0x0600"
+
+  [[ -d "${srcdir}"/build-${CARCH}-static ]] && rm -rf "${srcdir}"/build-${CARCH}-static
+  mkdir -p "${srcdir}"/build-${CARCH}-static && cd "${srcdir}"/build-${CARCH}-static
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=;-DCMAKE_PREFIX_PATH=" \
```

```suggestion
  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
```
I don't see `-DCMAKE_PREFIX_PATH` being used anywhere else, maybe this can be removed?

---

OK will try!